### DISCUSSION TITLE
Can not connect to mqtt broker with "MQTT Explorer" from macos

### DISCUSSION THREAD

**Discussion by TheSpirit** [teslamate-org/teslamate#3710] [Getting started] [answered]
HI,
i just set up teslamate, but i am unable to connect to the mqtt broker.
<img width="936" alt="Bildschirmfoto 2024-02-29 um 15 31 01" src="https://github.com/teslamate-org/teslamate/assets/6056918/5c7a466b-bbf6-4178-821d-1431422eee6f">
What do i have to fill in here?
Thanks

**Comment by tobiasehlert** [comment]
@TheSpirit, it depends a little on how you have setup your TeslaMate installation, but you should with the proper configuration be able to connect.

What setup are you running?
Do you want to share your _docker-compose.yaml_?

If you are using this: https://docs.teslamate.org/docs/installation/docker#instructions
Then you should use `mosquitto` in the _Host_ field and uncheck the _Validate certificate_ (unless you have added such things).

**Reply by TheSpirit** [reply, marked as answer]
Thank you. Works now just fine.
